cmd/statusd/main.go

```diff
@@ -33,6 +33,7 @@ var (
 	lesEnabled     = flag.Bool("les", false, "LES protocol enabled (default is disabled)")
 	whisperEnabled = flag.Bool("shh", false, "Whisper protocol enabled (default is disabled)")
 	swarmEnabled   = flag.Bool("swarm", false, "Swarm protocol enabled")
+	maxPeers       = flag.Int("maxpeers", 25, "maximum amount of p2p peers (includes all protocols)")
```

Comment:
Small gramma thing I believe: `peers` are countable so it should be `maximum number of p2p peers (including all protocols)`.